server/utils/assessments/tableUtils.ts

```diff
@@ -24,18 +25,24 @@ const getStatus = (assessment: AssessmentSummary): string => {
   return `<strong class="govuk-tag govuk-tag--grey">Not started</strong>`
 }
 
-const assessmentLink = (assessment: AssessmentSummary, linkText = '', hiddenText = ''): string => {
+const assessmentLink = (assessment: AssessmentSummary, person: FullPerson, linkText = '', hiddenText = ''): string => {
   return linkTo(
     paths.assessments.show,
     { id: assessment.id },
     {
-      text: linkText || assessment.person.name,
+      text: linkText || person.name,
       hiddenText,
       attributes: { 'data-cy-assessmentId': assessment.id, 'data-cy-applicationId': assessment.applicationId },
     },
   )
 }
 
+export const restrictedPersonCell = (person: RestrictedPerson) => {
+  return {
+    text: `${person.crn} is a LAO`,
```

Comment:
Should this be consistent with the booking table? So just "Limited access offender"? Or "LAO CRN: ${person.crn}", so it's consistent with applications?